week-1-ts-ecosystem/w01e01-typescript-basics/src/types.ts

```diff
@@ -0,0 +1,16 @@
+type AddressType = {
+    street: string;
+    no: number;
+}
+
+type UserType = {
+    name: string;
+    age: number;
+    email: string;
+    address: AddressType;
+};
+
+export type {
+    AddressType,
+    UserType,
+};
```

Comment:
`Address` moze być na ten moment zakryty - nie użwamy go nigdzie więc nie musi być `export` owany

Czemu przy typach stosujesz zapis `*Type` ? Jakie to ma uzasadnienie?

Reply:
Oj chyba taki nawyk z pracy. Korzyścią z takiego zapisu mogło by być uniknięcie kolizji nazw jakby była np. klasa User i typ User choć w praktyce taka sytuacja wydaje się mało prawdopodobna

Reply:
Mało prawdopodobna @DariuszGala z racji że `class` zawiera w sobie typ + dane